### ISSUE TITLE
Voting-cart staging (Django 1.9)

### ISSUE THREAD

**Issue by busla** [piratar/safetyvalve#13]
SV running on python3 and Django 1.9 on my staging server.
http://safetyvalve.projects.nonni.cc/

I merged the initial draft of the voting-cart as well (https://github.com/busla/safetyvalve/commits/feature/voting-cart) but it still needs a form and a view to handle the icekey signing. Not to mention the styling :-) 

@jamesrobb I´ll postpone the icekey signing until there is a conclusion on how you want to proceed.

Just noticed an issue with the fake_auth session handler on staging but it works. You just can´t flip it on/off. I´ll take a look at it later this week.

Still waiting on #safetyvalve | irc.freenode.net :-)

**Comment by busla** [comment]
naah, I just logged out and back in and now I can´t vote, so fake_auth is broken.

**Comment by busla** [comment]
removed the fake_auth check in base.html while I test and debug other stuff.

**Comment by jamesrobb** [comment]
Been open for too long - stale issue.
